quickget

```diff
@@ -3177,41 +3177,33 @@ function download_windows_workstation() {
         return $?
     }
 
-    # Extract everything after the last slash
-    local url_segment_parameter="${url##*/}"
+    local profile="606624d44113"
```

Comment:
What is this?

Reply:
Not entirely sure. It was merged into the [Fido](https://github.com/pbatard/Fido/issues/98) powershell script, I just reimplemented the same.